src/tui.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(clippy::non_ascii_literal)]
```

Comment:
Note to self: Move this back into `main.rs`, because the warning still shows up (probably because the lint is in a macro that gets expanded in `main`)